chap5/chap5-8/5-8-1/5-8-1.js

```
@@ -0,0 +1,80 @@
+  const [A, B] = input[i].split(" ").map((v) => parseInt(v) - 1);
+  G[A].push(B);
+  G[B].push(A);
+}
+
+const Q = {
+  d: Array(N),
+  h: 0,
+  t: 0,
+  len: () => Q.t - Q.h,
+  put: (v) => (Q.d[Q.t++] = v),
+  get: () => (Q.len() ? Q.d[Q.h++] : null),
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)